erpnext/templates/print_formats/includes/item_table_qty_swapped.html

```diff
@@ -0,0 +1,5 @@
+{{ doc.get_formatted("qty", doc) }}
```

Comment:
Instead of separate template you can use the if and else condition in the existing template